pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -141,6 +145,9 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig, adminKub
 	}
 
 	releaseImage := defaultReleaseImage
+	if len(PinnedReleaseImage) > 0 {
+		releaseImage = PinnedReleaseImage
```

Comment:
We can probably keep this section simple by making `defaultReleaseImage` itself a `var` and using:

```sh
if test -n "${OPENSHIFT_INSTALL_RELEASE_IMAGE}"
then
  LDFLAGS="${LDFLAGS} -X github.com/openshift/installer/pkg/asset/ignition/bootstrap.defaultReleaseImage=${OPENSHIFT_INSTALL_RELEASE_IMAGE}"
fi
```

in `hack/build.sh`.

Reply:
Done.